src/mobo_qm9.py

```diff
@@ -79,6 +84,26 @@ def get_surrogate_model(self, X, y):
         returns:
             model: Surrogate model for the MOBOQM9 model.
         """
+        X_scaled = Standardize(X)
```

Comment:
Thanks for the changes. Instead of using Standardize, can you use `input_transform` and `output_transform` in the `SingleTaskGP` model?